### ISSUE TITLE
Auto inititialization of TransportScheduler

### ISSUE THREAD

**Issue by SuhasHebbar** [prasadtalasila/TransportScheduler#41]
Presently an api call is needed to initialize the application. For auto initialization the code for initialization ( lines 24-33 in api.ex ) has to be moved to the init block of API.
But on doing so an api call is still required for initialization.

**Comment by prasadtalasila** [comment]
@SuhasHebbar please give reference to a code commit that has the problem. Only after that can @suhasspai can comment on the same.

**Comment by prasadtalasila** [comment]
Not a priority right now.
